packages/material-ui/src/Button/Button.spec.tsx

```diff
@@ -22,7 +22,7 @@ const ButtonTest = () => (
     <Button variant="outlined" aria-label="add">
       Outlined
     </Button>
-    <Button tabIndex={1} title="some button">
+    <Button tabIndex={-1} title="some button">
```

Comment:
We should disable jsx-a11y for .spec files like we to with .test files.

Reply:
Done